DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Imports:
 Suggests:
     knitr,
     rmarkdown,
+    RobinCar,
```

Comment:
may I ask if we can keep `RobinCar` out of the suggested packages? i.e. in the tests we use some static numbers. this is to avoid potential future complicated dependencies (in the recent meeting we are still discussing how we should harmonize between RobinCar and RobinCar2; keeping the independency can definitely help)